### DISCUSSION TITLE
Form step not showing

### DISCUSSION THREAD

**Discussion by laromana** [Budibase/budibase#6728] [General]
After the latest update I have noticed that the selected form step is not showing within the design are unless you refresh the page.

Example, if you are editing step one and want to now edit step 2 the component for step two won't show, it will keep showing step one in the design are unless you refresh the page.

**Comment by melohagan** [comment]
Hi @laromana 

Thanks for raising the bug: https://github.com/Budibase/budibase/issues/6729

As mentioned a fix should be in soon.
